handler/oauth2/flow_refresh.go

```diff
@@ -51,25 +51,31 @@ func (c *RefreshTokenGrantHandler) HandleTokenEndpointRequest(ctx context.Contex
 			return errorsx.WithStack(rErr)
 		}
 
-		return errorsx.WithStack(fosite.ErrInactiveToken.WithWrap(err).WithDebug(err.Error()))
+		return fosite.ErrInvalidGrant.WithWrap(err).
```

Comment:
Why remove the stack for all these errors?
Also, why change the explicit error here? Could this be breaking? From the hint it seems like this should still be `fosite.ErrInactiveToken`.

Reply:
OK, the spec is just weird here and `fosite.ErrInvalidGrant` is correct.

Reply:
> Why remove the stack for all these errors?

`WithWrap` uses the passed error to do the stack trace - so the internal error already has a stack trace and then we just re-use this. It's a bit like `fmt.Errorf("%w",err)`